src/main/scala/temple/utils/MonadUtils.scala

```diff
@@ -8,4 +8,11 @@ object MonadUtils {
     /** Reduces the Either to an item of the right type, by transforming a Left item if present */
     def fromEither(f: A => B): B = either.fold(f, identity)
   }
+
+  /** Adds the method `matchPartial` to any object, to call a partial function directly on it */
+  implicit class MatchPartial[A](x: A) {
```

Comment:
Ok so this is how it's defined, resolves my earlier comment but could we get a comment somewhere to point to MonadUtils?